.moai/reports/moai-local-self-evolution-design-20260804.html

```
@@ -0,0 +1,625 @@
+  <div class="stage">
+    <div class="stage-head"><span class="stage-num">6</span><h3>REMEMBER — 기억</h3></div>
+    <p>적용된 모든 제안은 <code>feedback_*.md</code> 주제 파일(category + 잘못된 패턴 + 올바른 접근 + WHY + surface 태그 + prediction + <code>verified:pending</code>)과 MEMORY.md 색인 한 줄을 쓴다 — Self-Refine에는 없고 ExpeL이 하중이라 증명한 교차-세션 저장. <strong>거부/되돌려진 제안도 <code>verified:false</code> 교훈을 쓴다</strong>(제안자가 같은 편집 부류를 재시도하지 않게). <code>memory_type</code> 라벨(CoALA)이 절차적 교훈은 <code>hns-*</code> 스킬로, 의미론적 사실은 <code>feedback_*.md</code>로 라우팅. 무장된 제안은 (재사용하는) <code>handoff/pending.json</code>을 타서 <code>/clear</code> 도중에도 다음 세션에서 이어진다.</p>
+    <dl>
+      <dt>서피스</dt><dd><code>memory/feedback_*.md</code> + <code>MEMORY.md</code> 색인 + <code>memory/_archive/</code>(cold tier, 삭제 불가) + <code>.moai/state/handoff/pending.json</code>(재사용).</dd>
+      <dt>도구</dt><dd>Claude Code 네이티브 auto-memory + <code>hns-lsel-applier</code>가 최종 apply 단계로 주제 파일 작성. MemGPT식 promote-on-retrieval-hit 페이징.</dd>
+    </dl>
+  </div>
+
+  <div class="stage">
+    <div class="stage-head"><span class="stage-num">7</span><h3>VERIFY — 검증</h3></div>
+    <p>적용 후 제안의 <code>verify_command</code> + <strong>의무 <code>/moai gate</code> 상위집합</code></strong>(lint+format+type+test)을 실행. 증거는 기존 verification-batch 계약대로 <code>.moai/state/verify/&lt;session&gt;/</code>에 지속(요약이 아니라 verbatim 출력 — verification-claim-integrity §2 귀속). 검증이 실패하면 <code>git revert lsel-&lt;proposal-id&gt;</code>로 자동 되돌리고, 제안의 <code>feedback_*.md</code>는 <code>verified:false</code> 표시되며 거부는 Reflexion 교훈이 된다. <strong>타임아웃 같은 불안정 신호에는 한 번 재시도</strong>하고 두 번째 비-타임아웃 실패에서만 되돌린다. 느린 주기로 임계치가 쌓이면(시계 아님) 세부 주제 파일 여러 개에서 추상 원칙 주제 파일 하나를 합성하고 원본은 cold tier로 내리는 정기 REFLECTION 패스.</p>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </code> ]

(tag-pair)